testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/OIDCClientRegistrationTest.java

```diff
@@ -50,10 +50,7 @@
 import java.util.*;
 import java.util.stream.Collectors;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static org.junit.Assert.*;
```

Comment:
Is it possible to remove star import and replace with the individual imports? We try to avoid star imports in java files in Keycloak. You can possibly change your IDE regarding this for the future.